src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -691,26 +679,12 @@ messages.imsg.m0064 = delete the other account
 messages.imsg.m0065 = Account merge initiated successfully
 messages.imsg.m0066 = you will be notified once it is completed
 messages.imsg.m0086 = Incorrect OTP. Number of attempts remaining : {remainingAttempt}
-messages.imsg.m0087 = Account merge not permitted
-messages.imsg.m0067 = Could not initiate the account merge. Click the Merge Account option in the Profile menu and try again
```

Comment:
messages.imsg.m0067 = Could not initiate the account merge. Click the Merge Account option in the Profile menu and try again
> In merge-account-status it is been used